shenyu-register-center/shenyu-register-client/shenyu-register-client-http/src/main/java/org/apache/shenyu/register/client/http/utils/OkHttpTools.java

```diff
@@ -99,16 +99,19 @@ public String post(final String url, final String json, final Headers headers) t
      * Get string.
      *
      * @param url   the url
-     * @param query the query
+     * @param userName the userName
+     * @param passWord the passWord
      * @return the http result
      * @throws IOException the io exception
      */
-    public String get(final String url, final Map<String, Object> query) throws IOException {
+    public String get(final String url, final String userName, final String passWord) throws IOException {
```

Comment:
Don't change it like that

Reply:
can add  another method  .   userName  ，passWord  maybe in Map

Reply:
Hi, @yu199195 
I modified this because only one method uses the get function. And this method is only used here and will not affect other methods. So I refactored the code like this.

Reply:
Hi ,@yu199195 
I modified this because only one method uses the get function. And this method is only used here and will not affect other methods.So I refactored the code like this.

Reply:
This method may later be called with multiple parameters